internal/rpc/starknet/types_test.go

```diff
@@ -0,0 +1,276 @@
+package starknet
+
+import (
+	"errors"
+	"io"
+	"reflect"
+	"testing"
+
+	"github.com/NethermindEth/juno/pkg/felt"
+	gocmp "github.com/google/go-cmp/cmp"
+
+	"gotest.tools/assert"
```

Comment:
Please use `"github.com/stretchr/testify/assert"`. It is more widely used.

Reply:
With `github.com/stretchr/testify/assert` we can't define custom comparisons and with `gotest.tools/assert` yes

Reply:
Ok sounds good. In future, then we should move away from using `testify/assert` and `testify/require` packages.